### ISSUE TITLE
Add support for Plex Tokens

### ISSUE THREAD

**Issue by Kaiede** [Kaiede/PlexPlaylistHost#2]
Attaching some of the details provided in an another issue. 

——

DefaultPrefs.json

```
{
"id": "plextoken",
"label": "Plex login token",
"type": "text",
"default": "my-token-here-that-I-found-from-viewing-xml-on-an-entity"
}
```

In init.py (under preferences)

```
PREFS__PLEXTOKEN = 'plextoken'
```

(and the updated GetPlexUrl function)

```
def GetPlexUrl(suffix=None):
plexUrl = 'http://%s:%s' %(Prefs[PREFS__PLEXIP], Prefs[PREFS__PLEXPORT])
if suffix is not None:
plexUrl += suffix

plexUrl += '?X-Plex-Token='
plexUrl += Prefs[PREFS__PLEXTOKEN]

Log.Debug('Requesting Plex URL: %s' % (plexUrl))
return plexUrl
```

**Comment by Kaiede** [comment]
It looks like this may be required to unblock Windows users of Plex.

**Comment by Kaiede** [comment]
Closing this out after tagging 1.0.3 with the merged PR.
